tests/common/Microsoft.DotNet.UpgradeAssistant.Abstractions.Tests/ProjectExtensionsTests.cs

```diff
@@ -85,5 +85,17 @@ public async Task CanConfigUpdaterApplyToMultipleLanguagesAsync(ProjectComponent
                 Assert.False(actual, "TestConfigUpdater only applies to C# and F# for ASP.NET Core");
             }
         }
+
+        [ApplicableComponents(ProjectComponents.AspNetCore)]
+        [ApplicableLanguage(Language.CSharp, Language.FSharp)]
+        private class TestObjectMultipleLanguage
+        {
+        }
+
+        [ApplicableComponents(ProjectComponents.AspNetCore)]
+        [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.FSharp, Name = "UHOH1 CodeFix Provider")]
+        private class TestCodeFixer
+        {
+        }
```

Comment:
Why empty classes?

Reply:
For test purposes - it verifies attributes extraction